docs/source/configuration/overview.mdx

```diff
@@ -742,6 +742,33 @@ query AliasedQuery {
 }
 ```
 
+<MinVersion version="1.50.0">
+
+### Extended reference reporting
+
+</MinVersion>
+
+<ExperimentalFeature />
+
+Apollo's legacy reporting didn't include input and enum references.
+Beginning in v1.50.0, you can configure the router to report input and enum usage.
+
+Configure extended reference reporting in `router.yaml` with the `telemetry.apollo.experimental_apollo_metrics_reference_mode` option like so:
+
+```yaml title="router.yaml"
+telemetry: 
+  apollo:
+    experimental_apollo_metrics_reference_mode: extended # Default is legacy
+```
+
+Once you enable this configuration, you can view enum and input usage along with other types of [field usage in GraphOS Studio](/graphos/metrics/field-usage).
+
+<Note>
+
+Once you enable `extended` reference reporting, there will be a lag in changes to metrics in Studio.
+This is because historical operations still use the `legacy` reference reporting.
```

Comment:
There shouldn't be a delay in metrics in Studio. But enabling this also allows us to run "enhanced checks" which is better at reducing the number of false-positives like the removal of unused enums and input fields, and there is a delay there because we can only run these enhanced checks if the flag has been enabled for the entire checks period. I can go into this further with you if you want.

Reply:
Ah, that makes sense, ty!

Reply:
They'll be automatic once the config has been turned on for the entire checks duration. As part of the check we run a query to check that the "enhanced reference reporting" flag has been turned on in usage reports, which happens when the flag is turned on.

Reply:
Also to clarify the "why", the new checks work mostly by detecting when enums or input values are not used, so we need to check that the setting is on for a while to be sure that we don't have the stats about them because they're actually not used, and not because they're just not included in "legacy" reports.